usr/src/lib/libzfs_core/common/libzfs_core.c

```diff
@@ -249,6 +249,31 @@ lzc_remap(const char *fsname)
 	return (error);
 }
 
+int
+lzc_rename(const char *source, const char *target)
+{
+	zfs_cmd_t zc = { 0 };
+	int error;
+
```

Comment:
Since this call (unfortunately) uses the old-style ioctl() call could you add the following assertions?
```
ASSERT3S(g_refcount, >, 0);
VERIFY3S(g_fd, !=, -1);
```

`lzc_exists()` (same situation - old way) and `lzc_ioctl()` (the new-style; common to most functions here)
both use these assertions to catch cases where consumers of the libzfs_core forget to call the init function before subsequent calls to the library.

Reply:
Thank you for the advice! I will do this.